test/testdrive/introspection-sources.td

```diff
@@ -282,15 +282,15 @@ mv1_primary_idx mv1
 
 > CREATE DEFAULT INDEX ii_arr ON vv_arr
 
-> SELECT records >= 300 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name='ii_arr' OR name='mv_arr'
-true
-true
+> SELECT records >= 300, size >= 300*96, capacity >= 300*96 FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name='ii_arr' OR name='mv_arr'
+true true true
+true true true
 
 # Test that non-arranging dataflows show up in `mz_dataflow_arrangement_sizes`
 
 > CREATE TABLE t3 (c int)
 
 > CREATE DEFAULT INDEX ii_empty ON t3
 
-> SELECT records, batches FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name='ii_empty'
-0 0
+> SELECT records, batches, size, capacity, allocations FROM mz_internal.mz_dataflow_arrangement_sizes WHERE name='ii_empty'
+0 0 0 0 0
```

Comment:
Would it be possible to also test that after dropping an arrangement, it entries disappear from `mz_arrangement_sizes` and/or related collections? I'd like to have something that ensures that we correctly retract the three new values on arrangement drops.

Reply:
Test added.